server/php-worker.cpp

```diff
@@ -32,6 +32,7 @@ double PhpWorker::enter_lifecycle() noexcept {
   }
   on_wakeup();
 
+  tvkprintf(php_code, 3, "PHP-worker lifecycle [req_id = %016llx]\n", req_id);
   paused = false;
   do {
     switch (state) {
```

Comment:
Maybe we'll move all logs about Worker state to this switch?

Reply:
add some state's details